docs/cqrs/pipeline/adding_custom_middlewares.md

```diff
@@ -0,0 +1,80 @@
+
+# Adding custom middlewares
+
+Custom middlewares can further augment the pipeline by employing the `UseMiddleware` method on `IApplicationBuilder`. To illustrate, consider the following basic example, which introduces a check if employee is blocked into the pipeline:
```

Comment:
We should point here to the ASP.NET Core docs on middlewares so that the users are aware, that this is a _normal_ middleware.